src/templates/installing/os/windows/nginx/install.md

```diff
@@ -1,8 +1,8 @@
 The following is an abbreviation of the [Windows nginx installation instructions](https://www.nginx.com/resources/wiki/start/topics/tutorials/install/#official-win32-binaries).
 
 1. Create a new directory `C:\nginx`
-2. [Download the stable nginx v{{versions.recommended.nginx}} win32 binary](http://nginx.org/en/download.html)
```

Comment:
Why remove the recommended nginx version? Maybe it would be better to automatically pull it from the internet like we do with the node version?

Reply:
See my initial comment on the PR.